crates/pop-contracts/src/utils/metadata.rs

```diff
@@ -131,11 +133,8 @@ fn get_contract_functions(
 /// # Arguments
 /// * `path` -  Location path of the project or contract artifact.
 /// * `message` - The label of the contract message.
-pub fn get_message<P>(path: P, message: &str) -> Result<ContractFunction, Error>
-where
-	P: AsRef<Path>,
-{
-	get_messages(path.as_ref())?
+pub fn get_message(path: &Path, message: &str) -> Result<ContractFunction, Error> {
```

Comment:
Curious about the motivation for this change, was it mainly for simplicity? The previous implementation accepted anything convertible to a `Path` (`&str`, `String`, `&Path`, `PathBuf`), which felt a bit more flexible.